lasagne/layers/dense.py

```diff
@@ -18,36 +18,38 @@ class DenseLayer(Layer):
     """
     A fully connected layer.
 
-    :parameters:
-        - incoming : a :class:`Layer` instance or a tuple
-            the layer feeding into this layer, or the expected input shape
-
-        - num_units : int
-            The number of units of the layer
-
-        - W : Theano shared variable, numpy array or callable
-            An initializer for the weights of the layer. If a Theano shared
-            variable is provided, it is used unchanged. If a numpy array is
-            provided, a shared variable is created and initialized with the
-            array. If a callable is provided, a shared variable is created and
-            the callable is called with the desired shape to generate suitable
-            initial values. The variable is then initialized with those values.
-
-        - b : Theano shared variable, numpy array, callable or None
-            An initializer for the biases of the layer. If a Theano shared
-            variable is provided, it is used unchanged. If a numpy array is
-            provided, a shared variable is created and initialized with the
-            array. If a callable is provided, a shared variable is created and
-            the callable is called with the desired shape to generate suitable
-            initial values. The variable is then initialized with those values.
-
-            If None is provided, the layer will have no biases.
-
-        - nonlinearity : callable or None
-            The nonlinearity that is applied to the layer activations. If None
-            is provided, the layer will be linear.
-
-    :usage:
+    Parameters
+    -----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape
+
+    num_units : int
+        The number of units of the layer
+
+    W : Theano shared variable, numpy array or callable
+        An initializer for the weights of the layer. If a Theano shared
```

Comment:
It's probably a good idea to reduce the copypasta here by referring to the Layer.create_param docs instead (as was done here: https://github.com/Lasagne/Lasagne/blob/master/lasagne/layers/conv.py#L141). This should probably also mention the expected shape of W, in case someone wants to provide an existing shared variable or a numpy array.